<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipientLogInClass</class>
 <widget class="QMainWindow" name="RecipientLogInClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blood Bank Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RecipientReg.qrc">
    <normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="backimg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/RecipientReg/bloodd.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="RecipientReg.qrc">:/RecipientReg/bloodd.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailTB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>290</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordTB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>290</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="mailalertmsg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordalertmsg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>331</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid rgb(177, 0, 3);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="wrong">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>Wrong mail or password</string>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label_3</zorder>
    <zorder>mailTB</zorder>
    <zorder>label_5</zorder>
    <zorder>passwordTB</zorder>
    <zorder>mailalertmsg</zorder>
    <zorder>passwordalertmsg</zorder>
    <zorder>wrong</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(177, 0, 3);
border-radius:25px;
color:white;
font: 700 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;
background-color:rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>  Recipient Log In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>521</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(57, 163, 255);
background-color:transparent;
border-radius:25px;
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>503</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/RecipientReg/pngwing.com (57).png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(117, 117, 117)</string>
    </property>
    <property name="text">
     <string>Previous Page</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RecipientReg.qrc"/>
 </resources>
 <connections/>
</ui>
